docs/topics/all-open-plugin.md

```diff
@@ -116,10 +116,15 @@ Or using the Gradle plugins DSL:
 
 ```groovy
 plugins {
-  id "org.jetbrains.kotlin.plugin.spring" version "%kotlinVersion%"
+  id "org.jetbrains.kotlin.plugin.spring" version "%kotlinSpringPluginVersion%"
```

Comment:
master already has these changes regarding the new variable. I pushed those changes in another PR and branch. The sole difference with master is the note on the line 123.